project/REVIEWING.md

```diff
@@ -0,0 +1,170 @@
+Pull request reviewing process
+==============================
+
+# Workflow
+
+A opened pull request can be in 1 of 5 distinct states, for each of which there is a corresponding
+label that needs to be applied:
+
+## Triage - `0-needs-triage`
+
+Maintainers are expected to triage new incoming pull requests by removing the `0-needs-triage`
+label and adding the correct labels (e.g. `1-design-review`) before any other interaction with the
+PR. The starting label may potentially skip some steps depending on the kind of pull request: use
+your best judgement.
+
+Checking for DCO should be done at this stage.
+
+Possible transitions from this state:
+
+ * Close: e.g., unresponsive contributor without DCO
+ * `1-needs-design-review`: general case
```

Comment:
yeah, thinking of this again, it helped me "get started", but I'm very much used to the order now, so probably moot.